pkg/grpc/server.go

```diff
@@ -130,9 +131,10 @@ func (s *server) StartNonBlocking() error {
 		listeners = append(listeners, l)
 	} else {
 		for _, apiListenAddress := range s.config.APIListenAddresses {
-			l, err := net.Listen("tcp", fmt.Sprintf("%s:%v", apiListenAddress, s.config.Port))
+			addr := apiListenAddress + ":" + strconv.Itoa(s.config.Port)
```

Comment:
should this be 
```suggestion
			addr := net.JoinHostPort(apiListenAddress, strconv.Itoa(s.config.Port))
```

Reply:
I changed it because it was failing with IPv6 addresses. With Dapr, we require IPv6 addresses to already be wrapped in `[]`, so this method was wrapping them again causing an error and Dapr wouldn't listen on IPv6